docs/csharp/delegate-class.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: System.Delegate and the `delegate` keyword
-description: Learn about the classes in the .NET Framework that support delegates and how those map to the 'delegate' keyword.
+description: Learn about the classes in .NET Framework that support delegates and how those map to the 'delegate' keyword.
```

Comment:
I removed "the", I feel the sentence is better with that. However, if you think this change isn't correct. Feel free to suggest reverting it.